### PR TITLE
Add Redis resource to docs

### PR DESCRIPTION
**Pull request by staceysalamon-aiven**
Add documentation for working with Redis.

### CODE REVIEW

File: docs/content/en/docs/resources/redis/_index.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+kind: Redis
+metadata:
+  name: redis-sample
+spec:
+  # gets the authentication token from the `aiven-token` Secret
+  authSecretRef:
+    name: aiven-token
+    key: token
+
+  # outputs the Redis connection on the `redis-connection` Secret
```
Comment (byashimov) [inline docs/content/en/docs/resources/redis/_index.md:26]:
```suggestion
  # outputs the Redis connection on the `redis-secret` Secret
```

File: docs/content/en/docs/resources/redis/_index.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  connInfoSecretTarget:
+    name: redis-secret
+
+  # add your Project name here
+  project: <your-project-name>
+
+  # cloud provider and plan of your choice
+  # you can check all of the possibilities here https://aiven.io/pricing
+  cloudName: google-europe-west1
+  plan: startup-2
```
Comment (byashimov) [inline docs/content/en/docs/resources/redis/_index.md:36]:
```suggestion
  plan: startup-4
```

File: docs/content/en/docs/resources/redis/_index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+```bash
+$ kubectl get redis.aiven.io redis-sample
+```
+
+The output is similar to the following:
+
+```bash
+NAME           PROJECT          REGION                PLAN        STATE
+redis-sample   <your-project>   google-europe-west1   startup-2   RUNNING
```
Comment (byashimov) [inline docs/content/en/docs/resources/redis/_index.md:63]:
```suggestion
redis-sample   <your-project>   google-europe-west1   startup-4   RUNNING
```

**Review by byashimov — CHANGES_REQUESTED** [review]

**Comment by byashimov** [comment]
LGTM, tested.

**Review by byashimov — APPROVED** [review]

File: docs/content/en/docs/resources/redis/_index.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    Reason:                Created
+    Status:                True
+    Type:                  Initialized
+    Last Transition Time:  2023-01-19T14:48:59Z
+    Message:               Instance was created or update on Aiven side, status remains unknown
+    Reason:                Created
+    Status:                Unknown
+    Type:                  Running
+  State:                   REBUILDING
+...
```
Comment (byashimov) [inline docs/content/en/docs/resources/redis/_index.md:76]: Replaced get command with describe

**Review by ivan-savciuc — APPROVED** [review]
LGTM

**Pull request merged**